app/api/user.py

```diff
@@ -247,10 +251,36 @@ def create_user(
         if supplier_code:
             user_data['role'] = 'supplier'
         else:
+            business_info_values = ''
+            organisation_name = ''
+            state = ''
+            postcode = ''
+            check_international = abn.validate(abn)
+            if check_international is not False:
+                try:
+                    # checks if international seller by checking if the nunmber provided is an ABN
```

Comment:
Haven't we passed the step where we check that the ABN is valid?  We now fetch the business details?